mkosi/distributions/debian.py

```diff
@@ -65,6 +65,7 @@ def tools_tree_packages(cls) -> list[str]:
             "systemd-container",
             "systemd",
             "tar",
+            "virtiofsd",
```

Comment:
This is only available in unstable/testing - and on the latter I wouldn't rely too much, given it's rust...

Reply:
Dropped from Debian for now